CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ All notable changes to KoalaBot will be documented in this file.
 A lot of these commands will only be available to administrators
 
 ## [Unreleased]
+- Fixed issues with permissions for a text channel resetting when a react for role is created.
```

Comment:
Please put under a rfr heading instead, basically to say that this is a change to the rfr cog as a bug fix

Reply:
at the moment it doesnt have a specific heading, just add the rfr heading above